packages/database/src/migrations/20210125042225-SamoaCovidSurveyHierarchyUpdate-modifies-data.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+import { updateValues } from '../utilities/migration';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const addressIdQuestionCode = 'QMIA029';
+const addressTextQuestionCode = 'QMIA010';
+const caseQuestionCode = 'QMIA025';
+const samoaEntityId = '5df1b88c61f76a485cd1ca09';
+const surveyResponseId = 'survey_response_id';
+
+async function getAnswersByQuestionCode(db, code) {
+  return db.runSql(`
+        select a.*
+        from answer a
+        inner join survey_response sr
+        on a.survey_response_id = sr.id
+        inner join question q
+        on a.question_id = q.id
+        where q.code in ('${code}');    
+  `);
+}
+
+async function findEntityIdByName(db, name) {
+  const cleanName = name.replace("'", '').trim();
+  if (!name.includes('/'))
```

Comment:
This check is not required: if a name includes unknown characters, an entity will not be found anyway

Reply:
Yeah It is confusing, this is for one special case mapping village name `V'ictoria` to `Victoria`(Can't remember the name so pick victoria)

Reply:
My comment was not about the `cleanName` function (this makes sense), it was about 

```javascript
if (!name.includes('/'))
```

Reply:
It will work as is though, so it's not important

Reply:
This is also an interesting one, if query with:
>   select * from entity
  where name = 'abc/cde'

It will confuse postgre sql and cause error.